api/docs/release.dox

```diff
@@ -241,6 +241,8 @@ Further non-compatibility-affecting changes include:
    trace analysis tool framework.
  - Added "-t drmemtrace" as the preferred launcher for the drmemtrace/drcachesim
    trace analysis tool framework.
+ - Added func_marker_filter_t filter to #dynamorio::drmemtrace::record_filter_t to filter
```

Comment:
Use # for a link for the func_marker_filter_t type.  Maybe should mention the CLI option name also (or instead).

Reply:
Done, also mentioned the CLI option "-filter_keep_func_ids".